src/frontend/src/lib/styles/tailwind/colors/theme-default.ts

```diff
@@ -0,0 +1,11 @@
+import { colorsBase } from './base';
+
+export const themeDefault = {
```

Comment:
see 
<img width="513" alt="image" src="https://github.com/user-attachments/assets/4577f7dd-cb5e-4f66-833e-544904d7c5ab">

Reply:
note that this will not be the actual theme, this is only declaring the available primitives

Reply:
so, this will be the theme default defined as Primitive, right? and a new theme will be for example `modernPrimitives` or `themePrimitivesModern`? or something similar?